boards/samr30-xpro/include/board.h

```diff
@@ -36,7 +36,11 @@ extern "C" {
 #define AT86RF2XX_PARAM_INT         GPIO_PIN(PB, 0)
 #define AT86RF2XX_PARAM_SLEEP       GPIO_PIN(PA, 20)
 #define AT86RF2XX_PARAM_RESET       GPIO_PIN(PB, 15)
+#if CLOCK_CORECLOCK >= 20000000U || defined(DOXYGEN)
```

Comment:
CLOCK_CORECLOCK is not defined unless you add `#include "periph_conf.h"` after `#include "cpu.h"`.